src/main/external-resources/renderers/Philips-PFL.conf

```diff
@@ -155,34 +147,26 @@ KeepAspectRatio = true
-# DTS is not supported.
-# MPEG-TS with H.264 is not supported.
-# 6 (5.1) audio channels are supported.
-# H.264 within AVI containers is not supported.
-# DivX with QPEL within AVI container is not supported.
-# GMC is supported.
-# WMV is supported, but not with 5.1 or PRO audio.
-# WMA audio is only supported within a WMA or WMV container.
-# LPCM audio is only supported with MPEG-2, AIFF or WAV.
 # MOV is not supported well enough to confidently stream.
-# MKV with VC-1, MPEG-2, MPEG-4 or DivX is not supported.
-
 # Supported video formats:
-Supported = f:mpegps   v:mpeg1       a:mpa|mp3                          m:video/mpeg
-Supported = f:mpegps   v:mpeg2       a:ac3|lpcm|mp3|mpa                 m:video/mpeg
+Supported = f:mpegps   v:mpeg1       a:mpa|mp3
+Supported = f:mpegps   v:mpeg2       a:ac3|lpcm|mp3|mpa
 Supported = f:mpegts   v:mpeg2       a:mpa|ac3|lpcm                     m:video/vnd.dlna.mpeg-tts
 Supported = f:avi      v:mp4         a:ac3|mp3|mpa                      m:video/x-msvideo
 Supported = f:avi      v:divx        a:ac3|mp3|mpa                      m:video/x-msvideo               qpel:no
 Supported = f:divx     v:divx        a:ac3|mp3|mpa                      m:video/x-msvideo
 Supported = f:mp4      v:h264        a:aac-lc|he-aac|ac3|mp3|mpa        m:video/h264                    b:4528690
 Supported = f:mp4      v:mp4         a:aac-lc|he-aac|ac3|mp3|mpa        m:video/h264
-Supported = f:wmv      v:vc1         a:wma                              m:video/x-ms-wmv                n:2          s:48000
-Supported = f:mkv      v:h264        a:aac-lc|he-aac|ac3|mp3|mpa        m:video/x-matroska
+Supported = f:wmv      v:vc1         a:wma                        n:2          s:48000
+Supported = f:mkv      v:h264        a:aac-lc|he-aac|ac3|mp3|mpa
 
 # Supported audio formats:
-Supported = f:adts       n:2   a:aac-lc s:48000               m:audio/vnd.dlna.adts
-Supported = f:aiff|wav   n:2   a:lpcm   s:44100   b:1445069   m:audio/L16
-Supported = f:mp3|mpa    n:2            s:48000   b:327680    m:audio/mpeg
-Supported = f:mp4        n:6   a:aac-lc s:48000   b:327680    m:audio/mp4
-Supported = f:wma        n:2   a:wma    s:48000   b:348160    m:audio/x-ms-wma
+Supported = f:adts       n:2     a:aac-lc|he-aac   s:48000
+Supported = f:aiff|wav   n:1     a:lpcm            s:44100     b:1445069       m:audio/L16;rate=44100;channels=1
+Supported = f:aiff|wav   n:2     a:lpcm            s:44100     b:1445069       m:audio/L16;rate=44100;channels=2
+Supported = f:aiff|wav   n:1     a:lpcm            s:48000     b:1445069       m:audio/L16;rate=48000;channels=1
+Supported = f:aiff|wav   n:2     a:lpcm            s:48000     b:1445069       m:audio/L16;rate=48000;channels=2
+Supported = f:mp3|mpa    n:2                       s:48000     b:327680
+Supported = f:m4a        n:6     a:aac-lc|he-aac   s:48000     b:327680        m:audio/mp4
+Supported = f:wav        n:2     a:mp3             s:48000     b:1445069
```

Comment:
https://github.com/UniversalMediaServer/UniversalMediaServer/blob/master/src/main/external-resources/renderers/Philips-PFL.conf#L129
So it seem that some comments are wrong and unecessary.